cmd/rookflex/cmd/expander.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/rook/rook/pkg/daemon/ceph/agent/flexvolume"
+	"github.com/spf13/cobra"
+	"os/exec"
+	"strconv"
+)
+
+var (
+	expandFSCmd = &cobra.Command{
+		Use:   "expandfs",
+		Short: "Expands the size of pod filesystem",
+		RunE:  handleExpandFs,
+	}
+)
+
+func init() {
+	RootCmd.AddCommand(expandFSCmd)
+}
+
+func handleExpandFs(cmd *cobra.Command, args []string) error {
+	var opts = &flexvolume.ExpandOptions{}
+
+	err := json.Unmarshal([]byte(args[0]), opts)
+	if err != nil {
+		return fmt.Errorf(
+			"Rook: Could not parse options for expand %s. Got %v",
```

Comment:
Use `%+v` instead

Reply:
Rook doesn't use the pattern of prefixing errors with `Rook: `. Please remove this where it appears.

Reply:
I think I copy & paste it from somewere in the repository. Sorry :) I will get rid of it.